src/network/networkd-network.c

```diff
@@ -607,14 +607,17 @@ int config_parse_netdev(const char *unit,
 
         switch (kind) {
         case NETDEV_KIND_BRIDGE:
+                network->bridge = netdev_unref(network->bridge);
```

Comment:
It's more of a question :-) I'm wondering if it's possible to trigger the leak in practice. I can't seem to find a way to do it easily. The commit message says that the leak is possible but how possible is it?

Reply:
Yeah, fortunately, missing `netdev_unref()` here does not cause memleak, as `manager_free()` uses
```
        while ((netdev = hashmap_first(m->netdevs)))
                netdev_unref(netdev);
```
instead of 
```
        while ((netdev = hashmap_steal_first(m->netdevs)))
                netdev_unref(netdev);
```
But, without this fix, actually reference counter of the netdev increases with the number of `Bridge=` lines.